OSXvnc-server/main.c

```diff
@@ -656,6 +656,7 @@ void rfbGetFramebufferUpdateInRect(int x, int y, int w, int h) {
             CGContextDrawImage(context, CGRectMake(0, 0, w, h), image);
             imageRef = CGBitmapContextCreateImage(context);
             CGContextRelease(context);
+            CGImageRelease(image);
```

Comment:
I suggest to release `image` as early as possible. Would it work two lines earlier, too?

Reply:
done